src/test/resources/jmx.yaml

```diff
@@ -1,7 +1,7 @@
 init_config:
 
 instances:
-    -   process_name_regex: .*surefire.*
+    -   jvm_direct: true
```

Comment:
Aren't `process_name_regex` and `jvm_direct` slightly different? One will attach to a running JVM, while the other is launched as a `-javaagent` with the application? Would different test cases be needed?

Reply:
They are slightly different, but from this test perspective, they're mostly the same.  There are plenty of other tests that verify the `process_name_regex` behavior, so I felt it was ok to change this one rather than add additional complexity in the tests.

(If we moved the all to `jvm_direct` we might be able to run the tests from within an IDE, rather than just via maven.)